Unhandled exception in the UI thread. (Widget is disposed)

GAPID Version: 0.9.0:0:developer
OS: linux amd64

Clicking on link in command tree:
```
org.eclipse.swt.SWTException: Failed to execute runnable (org.eclipse.swt.SWTException: Widget is disposed)
	at org.eclipse.swt.SWT.error(Unknown Source)
	at org.eclipse.swt.SWT.error(Unknown Source)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Unknown Source)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Unknown Source)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Unknown Source)
	at org.eclipse.jface.window.Window.runEventLoop(Window.java:818)
	at org.eclipse.jface.window.Window.open(Window.java:794)
	at com.google.gapid.Main$UI.show(Main.java:109)
	at com.google.gapid.Main.main(Main.java:67)
Caused by: org.eclipse.swt.SWTException: Widget is disposed
	at org.eclipse.swt.SWT.error(Unknown Source)
	at org.eclipse.swt.SWT.error(Unknown Source)
	at org.eclipse.swt.SWT.error(Unknown Source)
	at org.eclipse.swt.widgets.Widget.error(Unknown Source)
	at org.eclipse.swt.widgets.Widget.checkWidget(Unknown Source)
	at org.eclipse.swt.widgets.Widget.getData(Unknown Source)
	at com.google.gapid.widgets.LinkifiedTree$LabelProvider.getLabelNoUpdate(LinkifiedTree.java:464)
	at com.google.gapid.widgets.LinkifiedTree$LabelProvider.update(LinkifiedTree.java:382)
	at com.google.gapid.widgets.LinkifiedTree$LabelProvider.lambda$onShow$0(LinkifiedTree.java:285)
	at com.google.gapid.models.ApiState$3.onUiThread(ApiState.java:136)
	at com.google.gapid.models.ApiState$3.onUiThread(ApiState.java:127)
	at com.google.gapid.rpc.UiCallback.lambda$onFinish$0(UiCallback.java:46)
	at com.google.gapid.widgets.Widgets.ifNotDisposed(Widgets.java:119)
	at com.google.gapid.widgets.Widgets.lambda$scheduleIfNotDisposed$1(Widgets.java:147)
	at com.google.gapid.widgets.Widgets.lambda$schedule$0(Widgets.java:139)
	at org.eclipse.swt.widgets.RunnableLock.run(Unknown Source)
	... 7 more
```

--- a/gapic/src/main/com/google/gapid/widgets/LinkifiedTree.java
+++ b/gapic/src/main/com/google/gapid/widgets/LinkifiedTree.java
@@ -286,8 +286,10 @@ public abstract class LinkifiedTree<T, F> extends Composite {
     public void onShow(TreeItem item) {
       T element = getElement(item);
       contentProvider.load(element, () -> {
-        update(item);
-        refresher.refresh();
+        if (!item.isDisposed()) {
+          update(item);
+          refresher.refresh();
+        }
       });
     }
 

